pyiron_base/generic/datacontainer.py

```diff
@@ -905,10 +906,23 @@ def write(self, file_name):
         """
         write(self.to_builtin(), file_name)
 
+    def _force_load(self, recursive=True):
+        """
+        Load all HDFStubs present in the data container.
+
+        Args:
+            recursive (bool): force also nested data containers, default True
+        """
+        if not self._lazy: return
+
+        # values are loaded from HDF once they are accessed via __getitem__, which is implicetly called by values()
+        for v in self.values():
+            if recursive and isinstance(v, DataContainer):
```

Comment:
Do we have other things which are lazy? If yes, they should also be `_force_load()`ed.

Reply:
So far only `DataContainers` (and subclasses) do this, so should be ok.